### PR TITLE
Update Solana libraries to 1.13.6

### PR DESCRIPTION
**Pull request by alex-at-planetariummusic**

### CODE REVIEW

File: bot/src/rpc_client_utils.rs [hunk 1]
```diff
@@ -36,6 +36,47 @@ fn new_spinner_progress_bar() -> ProgressBar {
...
+fn new_tpu_client_with_retry(
+    rpc_client: &Arc<RpcClient>,
+    websocket_url: &str,
+) -> Result<TpuClient, TpuSenderError> {
+    use solana_client::{
+        client_error::{ClientError, ClientErrorKind},
+        rpc_request::RpcError,
+    };
+    let mut retries = 128; // connecting with a 32-slot epoch can sometimes take awhile
+    let sleep_ms = 200;
```
Comment (joncinque) [inline bot/src/rpc_client_utils.rs:49]: I took another look at this, and the errors are still with the leader schedule bit, can you try bumping this sleep up?
```suggestion
    let sleep_ms = 400;
```
If it's still no good, we might have to increase the retries by another 64.

Side note, I tried upgrading straight to 1.14, but it makes things even worse with the minimum stake delegation. I can help with that once the new stake pool program is cut and released.

Comment (alex-at-planetariummusic) [reply bot/src/rpc_client_utils.rs]: Thanks; I've tried all sorts of combinations of sleeps/retries, and nothing seems to work consistently. So it seems like the options are either to rewrite (if possible) the test, or disable it.

Comment (alex-at-planetariummusic) [reply bot/src/rpc_client_utils.rs]: And... I believe the error we're getting now is different from before:
```

Waiting for fees to stabilize 1...
Waiting for fees to stabilize 2...
thread 'stake_pool_v0::test::this_test_is_too_big_and_slow' panicked at 'called `Result::unwrap()` on an `Err` value: "Failed to create validator stake accounts"', bot/src/stake_pool_v0.rs:1055:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by alex-at-planetariummusic** [comment]
Closing; will create a new branch rebased off of master

**Pull request closed without merging**